### ISSUE TITLE
Feature request: dask book

### ISSUE THREAD

**Issue by raybellwaves** [dask/community#71]
This came up in the chat of the dask tutorial at SciPy 2020.

I think it's a great idea. Mostly because I love reading books on planes. There is something nice about slowly reading through code on paper to understand how the API works etc.

The FastAI book could offer a good template https://github.com/fastai/fastbook and is a good supplement to courses/videos/tutorial/blog etc.

I believe there are a ton of spark books e.g. https://www.oreilly.com/library/view/learning-spark-2nd/9781492050032/. A dask book could take inspiration from these.

**Comment by jacobtomlinson** [comment]
There is a manning one.

https://www.manning.com/books/data-science-with-python-and-dask

**Comment by mrocklin** [comment]
I think that a Dask book would be great!  As long as I don't have to write
it :)

**Comment by jcrist** [comment]
> As long as I don't have to write it

Agreed. From personal experience, Packt, O'Reilly, and Manning are more than eager to get someone to write a Dask book, I just don't want to do it.

**Comment by raybellwaves** [comment]
![Screenshot from 2020-09-07 15-31-04](https://user-images.githubusercontent.com/17162724/92414410-1ca55200-f122-11ea-993e-b8a087e94152.png)

hehe

**Comment by jacobtomlinson** [comment]
I wonder if Packt are using the Dask org membership to reach out to people. We added @raybellwaves recently for example.

**Comment by mrocklin** [comment]
Some employees of Coiled who are not part of the Dask org were also
targeted :)
